labcodes/django-react-webpack
# Config cypress

fix #8 

**Link to task:** #8

**How to test:**
- reinstall node depencies;
- run `make run_e2e`;
- click on "Run all test suites";
- make sure that all tests are passing.

**Description of your solution:**
We installed Cypress as a new node dependency and added sample tests for the two react pages.
